pbs.py

```diff
@@ -256,10 +256,10 @@ def read_pbsnodes_yaml(fn, write_method):
     return pbs_nodes
 
 
-def map_pbsnodes_to_wn_dicts(node_dict, pbs_nodes):
-    for (pbs_node, (idx, cur_node_nr)) in zip(pbs_nodes, enumerate(node_dict['wn_list'])):
-        node_dict['wn_dict'][cur_node_nr] = pbs_node
-        node_dict['wn_dict_remapped'][idx] = pbs_node
+def map_pbsnodes_to_wn_dicts(cluster_dict, pbs_nodes):
+    for (pbs_node, (idx, cur_node_nr)) in zip(pbs_nodes, enumerate(cluster_dict['wn_list'])):

```

Comment:
what about renaming it to clsuter_dictionary ? code is written once but read many times over ... consider it